release-notes.txt

```diff
@@ -0,0 +1,38 @@
+Release v0.7.1 release notes.
+
+New features:
+    Add alert manager, define several rules for alert. Operator will receive email notification if cluster has some problem.
```

Comment:
Do we have instruction for how to set the rules and update the email notification? 
Do we have instruction for problems types and error codes?
We can add these in here as reference links, or add few screenshot to brief the feature here.